.github/workflows/build.yaml

```diff
@@ -39,6 +39,8 @@ jobs:
             python-version: "3.11"
           - os: macos-latest
             python-version: "3.11"
+          - os: ubuntu-latest
+            python-version: "3.12.0rc1"
```

Comment:
You can fold this directly into the `python-version` list above as simply `"3.12"`.

Then, in the `with:` property of the `setup-python` action, you can add:

```yaml
allow-prereleases: true
```

These lines are not in scope for this diff so I cannot leave comments or suggestions. :cry:

Reply:
Thanks for the pointers!

My intent was to give us only one 3.12 build -- and my linux-first bias was at play.
But I don't mind setting up all 3 platforms, since it might catch things, so I'll go with your suggestion.

I'll also set `allow-prereleases` and we'll see if that does the trick for a nice clean build.

Reply:
Putting `"3.12"` into the list of versions above would only add it for Linux, wouldn't it?

Reply:
_Looks at the build matrix again._

Yes. Yes it would.

...

Some projects have bigger default matrices, but we're just using `include` for our non-linux builds. I forgot.

Well, anyway! We can add more builds to the matrix whenever we want, so we can always debate more builds in the future. :smile: